src/options/options.html

```diff
@@ -164,6 +164,15 @@ <h2 data-t="long_break_title"></h2>
               </div>
             </fieldset>
           </div>
+          <div class="section">
+            <h2 data-t="startup_title"></h2>
```

Comment:
It would be helpful to add a short message here explaining what autostart is to the user, something along the lines of "...Automatically start a Pomodoro at this time...".